docs/pages/serving_an_origin.mdx

```diff
@@ -55,123 +57,309 @@ Once you go through the process, locate your credential files and set the follow
 
 Since your TLS certificate is associated with your domain name, you will need to change the default hostname of Pelican server to be consistent. Set `Server.Hostname` to your domain name (e.g. `example.com`).
 
-## Serve an Origin
+## Serving Origins
```

Comment:
```suggestion
## Serve Origins
```

I thought we want to keep titles/subtitles started with a verb ( from Brian Lin's instructions on written a technical docs)

Reply:
I think this still aligns with the style formatting Brian prefers. Take a look at the current OSDF documentation he and Matyas have written -- there they also begin headers with gerunds like "Running", "Configuring", "Providing" etc.